app/views/layouts/application.html.erb

```diff
@@ -76,11 +76,13 @@
       <%= render 'shared/phase_banner' if ENV['NOTICE_BANNER'] %>
       <%= yield :before_main_content %>
 
+      <%= render 'shared/top_breadcrumbs' %>
+      <%= render 'shared/switch_banner' %>
```

Comment:
Those partials have been extracted from the `shared/search` partial.
(why did the search contain the breadcrumb and switch? 🤷 )

Reply:
So much better